samples/communities/communities.json

```diff
@@ -4,10 +4,10 @@
       "application": "Communities Customization",
       "payload": {
         "include-files": [
-          "communities/communitiesCustomization.css"
+          "communities/communitiesCustomization.js"
```

Comment:
i think you are missing the "samples" folder here in the path
"samples/communities/communitiesCustomization.js"

also, you can just point to the css directly so try 
"samples/communities/communitiesCustomization.css"